core/src/test/scala/org/apache/spark/scheduler/OutputCommitCoordinatorIntegrationSuite.scala

```diff
@@ -46,21 +45,21 @@ class OutputCommitCoordinatorIntegrationSuite
   }
 
   test("exception thrown in OutputCommitter.commitTask()") {
-    // Regression test for SPARK-10381
-    failAfter(Span(60, Seconds)) {
+    val e = intercept[SparkException] {
       withTempDir { tempDir =>
         sc.parallelize(1 to 4, 2).map(_.toString).saveAsTextFile(tempDir.getAbsolutePath + "/out")
       }
-    }
+    }.getCause.getMessage
+    assert(e.endsWith("failed; but task commit success, data duplication may happen."))
   }
 }
 
 private class ThrowExceptionOnFirstAttemptOutputCommitter extends FileOutputCommitter {
   override def commitTask(context: TaskAttemptContext): Unit = {
     val ctx = TaskContext.get()
+    super.commitTask(context)
```

Comment:
why this change? I think the original test wanted to test the case that we failed before committing in the first task attempt.

Reply:
revert now